code/src/components/styling/AlbumName.css

```diff
@@ -0,0 +1,10 @@
+.album-name {
```

Comment:
Where these css files for specific components used in the end? If not - perhaps you could have removed them before you made the pull request?